src/vshaxe/Main.hx

```diff
@@ -109,19 +109,19 @@ class Main {
 	}
 
 	function setLanguageConfiguration():Void {
+		var type:String = workspace.getConfiguration("haxe").get("extendedIndentation", "dynamic");
```

Comment:
We can't just check settings once at startup and call it a day, settings can change while the extension is running. There's an `onDidChangeConfiguration` event we use in various places, for instance: https://github.com/vshaxe/vshaxe/blob/8a78fbef8616f23b28d1cb5a4efebb0babbf098b/src/vshaxe/view/dependencies/DependencyTreeView.hx#L55